google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/datacatalog_cli.py

```diff
@@ -55,9 +56,28 @@ def _get_metadata_scraper(self):
     def _get_host_arg(self, args):
         return args.kafka_host
 
-    def _get_connection_config(self, args):
-        group_id = args.group_id or 'kafka2dc'
-        return {'bootstrap.servers': args.kafka_host, 'group.id': group_id}
+    def _get_schema_registry_connection_config(self, args):
```

Comment:
can we extract the schema registry support to a separate file / class?

Reply:
This method only deals with command line arguments: composing a dictionary out of those that are provided. In this sense, I think that having it in the DatacatologCLI class actually makes sense.